pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1023,6 +1023,8 @@ type WorkflowStatus struct {
 	// Time at which this workflow completed
 	FinishedAt metav1.Time `json:"finishedAt,omitempty" protobuf:"bytes,3,opt,name=finishedAt"`
 
+	EstimatedDuration time.Duration `json:"estimatedDuration,omitempty" protobuf:"varint,16,opt,name=estimatedDuration,casttype=time.Duration"`
```

Comment:
* [ ] add comment
* [ ] currently this is `time.Duration`, i.e. would it be better to be seconds or ms?